src/Components/Publishing/Header/__test__/__snapshots__/FeatureHeader.test.tsx.snap

```diff
@@ -2916,8 +2916,8 @@ exports[`feature renders superArticle full header properly 1`] = `
   -webkit-box-align: center;
   -ms-flex-align: center;
   align-items: center;
-  z-index: 100;
-  padding: 50px;
+  z-index: 1;
+  padding: 50px 45px 50px;
```

Comment:
Not sure why this is in here, since it relates to my last pr, but the new version is correct.